### PR TITLE
Add Slovenian translation

### PR DESCRIPTION
**Pull request by filips123**
Add Slovenian translation.

### CODE REVIEW

File: src/locales.js [hunk 1]
```diff
@@ -60,6 +60,16 @@ export const nb_NO = {
   LBL_LOG_OUT: 'Logg Ut'
 };
 
+export const en_EN = {
```
Comment (frju365) [inline src/locales.js]: better perhaps "sl_SI" ?

Comment (filips123) [reply src/locales.js]: @andersevenrud @frju365 I fixed this.

**Comment by andersevenrud** [comment]
Seems like the Codacy job is stuck or something -- but everything looks OK :+1:

**Comment by andersevenrud** [comment]
Thanks!

**Pull request merged**